buildkite/bazelci.py

```diff
@@ -1670,8 +1677,13 @@ def main(argv=None):
                                    use_but=args.use_but,
                                    incompatible_flags=args.incompatible_flag)
         elif args.subparsers_name == "runner":
-            configs = fetch_configs(args.http_config, args.file_config)
-            execute_commands(config=configs.get("platforms", None)[args.platform],
+            tasks = get_tasks(args)
+            # there can be multiple tasks for a platform
+            for name in tasks:
+                task = tasks[name]
+                if hasattr(task, "platform") and task.platform != args.platform:
+                    continue
+                execute_commands(config=task,
```

Comment:
If I understand this correctly, this code intends to run all tasks from a given platform one after the other on the same machine?

I think it would be better to generate individual CI jobs for each task in `print_project_pipeline`, so that they cannot influence each other and run in parallel on all available machines.

WDYT?

Reply:
Fixed, did my best :) So bad I can not run it,